test/framework/util.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+)
+
+func Logf(format string, a ...interface{}) {
```

Comment:
I'm debated on exposing this as a public method, first because logging is tangential to the goal to the framework, second because in the long term the goal is to make the framework to work without ginkgo & gomega: see
https://github.com/kubernetes-sigs/cluster-api/issues/2306#issuecomment-617002275
https://github.com/kubernetes-sigs/cluster-api/issues/2955

Reply:
made the changes @fabianofranz let me know what do you think when you have time :)